packages/cli/test/utils/simulation/beacon_clients/lighthouse.ts

```diff
@@ -92,10 +91,9 @@ export const generateLighthouseBeaconNode: BeaconNodeGenerator<BeaconClient.Ligh
       },
       health: async () => {
         try {
-          await got.get(`http://127.0.0.1:${ports.beacon.httpPort}/eth/v1/node/health`);
-          return {ok: true};
+          return await fetch(`http://127.0.0.1:${ports.beacon.httpPort}/eth/v1/node/health`);
         } catch (err) {
-          if (err instanceof RequestError && err.code !== "ECONNREFUSED") {
+          if (err instanceof FetchError && err.code !== "ECONNREFUSED") {
             return {ok: true};
           }
```

Comment:
This logic is very strange, @nazarhussain why do we return health status as `{ok: true}` if we encountered an error? Doesn't seem right to be

Reply:
If I remember correctly `/node/health` returns status other than 200 in case of syncing. So as long as the fetch detects the port is listening, we are good to return `ok: true`.

Reply:
> If I remember correctly /node/health returns status other than 200 in case of syncing and that is considered an error in request handling

Maybe that was some weird request handling that `got` had but the [getHealth](https://ethereum.github.io/beacon-APIs/#/Node/getHealth) API either returns a 200 (synced) or 206 (syncing) which both should not be considered as an error.

Either way, now with `fetch`, we don't need to worry about that as it is handled correctly 200/206 will both result in `ok: true` and network errors will throw and other http errors (400-599) will result in `ok: false` which is what we want if the node returns 503 meaning "Node not initialized or having issues".

Reply:
So, May I delete?  ```TypeScript
          if (err instanceof FetchError && err.code !== "ECONNREFUSED") {
            return {ok: true};
          }
```

Reply:
yeah, exactly, I don't think we need any of that, it also quite confusing

Reply:
just keep in mind, fetch will still throw an error if it's related to an network issue, so we still need the try/catch